### PR TITLE
Food form

### PR DESCRIPTION
**Pull request by izaxveta**
#### Pivotal URL:
https://www.pivotaltracker.com/story/show/154615157

#### What does this PR do?
This PR adds the ability for a user to fill in a form to add a new food item. A post request is made to the API, and the response data from the API is then appended to the list of all food items. Page is not refreshed.

#### Where should the reviewer start?
Start in the foods.html to inspect the form, and continue in the foods.js to see how functionality interacts with form input objects.

#### Any additional context you want to provide?
Styling has not been added and will be dealt with at a later time.

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: foods.html [hunk 1]
```diff
@@ -5,6 +5,17 @@
     <title>Quantified Self - Manage Foods</title>
   </head>
   <body>
+    <form id="new-food">
```
Comment (marshallhouston) [inline foods.html]: I think this should be more specific with the id as `new-food-form`

File: lib/index.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 var foodsRequests = require('./requests/foods')
-require("./foods")
+require('./foods')
```
Comment (marshallhouston) [inline lib/index.js:2]: I appreciate the detailed focus on consistency about quotations!

**Review by marshallhouston — CHANGES_REQUESTED** [review]
I've made a small request. Once this is changed, I'll approve!

**Comment by izaxveta** [comment]
Changes made to form id naming.

**Review by marshallhouston — APPROVED** [review]

**Pull request merged**